src/operun/crm/ldap/templates/ldap_sync.pt

```diff
@@ -14,14 +14,29 @@
       <div>
 
         <h1 i18n:translate="ldap_sync_title">LDAP Sync</h1>
-
-        <h3 i18n:translate="ldap_sync_message">TODO</h3>
-
-        <form action="${context/absolute_url}/@@ldap-sync"
-              id="ldap-sync"
-              name="ldap-sync-form"
-              enctype="multipart/form-data"
-              method="post">
+        <h3 i18n:translate="ldap_sync_message">Proceeding will attempt to:</h3>
```

Comment:
WTF? Pleas drop this... I tried several times to understand. No chance. Let's first discuss what exactly should happen. Then we try to explain it.